students/cindywaldron/lesson3/list_lab.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+a_list = [ "Apples", "Pears", "Oranges", "Peaches"]
+
+print(a_list)
+# ask user to enter a fruit
+response = input("Enter a fruit name > ")
+#display user's input
+print("You entered: " + response)
+print(response + " is added to end of list")
+# add the input to end of the list
+a_list.append(response)
+# display the list
+print(a_list)
+# ask user to enter a number
+num = int(input("Enter a number > "))
+
+#display the item
+print(a_list[num-1])
+# Enter another fruit
+response = input("Enter another fruit name > ")
+a_list2 = [response]
+print(response + " is added to front of list. New list ==>")
+# add the fruit to front of the list
+print(a_list2 + a_list)
+# enter another fruit
+response = input("Enter another fruit name > ")
+# insert an item to front of the list
+a_list.insert(0, response)
+print("Inserted " + response + " to front of list.")
+#display the list
+print(a_list)
+# print fruit name starts with 'P'
+for item in a_list:
+    if 'P' in item:
```

Comment:
it is best to ensure you're only checking for first letter like `item.startswith('P')` or `item[0] == 'P'`